extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/devmode/console/DevConsole.java

```diff
@@ -35,8 +35,13 @@
     final Engine engine;
     final Map<String, Map<String, Object>> extensions = new HashMap<>();
 
-    DevConsole(Engine engine) {
+    final Map<String, Object> globalData = new HashMap<>();
+
+    DevConsole(Engine engine, String httpRootPath, String frameworkRootPath) {
         this.engine = engine;
+        this.globalData.put("httpRootPath", httpRootPath);
+        this.globalData.put("frameworkRootPath", frameworkRootPath);
```

Comment:
Should this be named "nonApplicationRootPath" instead of "frameworkRootPath" to align with the name elsewhere?